lib/openhab/dsl.rb

```diff
@@ -603,11 +640,10 @@ def store_states(*items)
     #   end
     #
     def ensure_states
-      old = Thread.current[:openhab_ensure_states]
-      Thread.current[:openhab_ensure_states] = true
+      old = ensure_states!
```

Comment:
:sobeautiful: 

I have taught you well, padawan!

Reply:
s/padawan/cadet/